neuralpp/symbolic/sympy_expression.py

```diff
@@ -275,7 +296,7 @@ def new_quantifier_expression(
         # Unlike for FunctionApplication where the non-constructable is the exception,
         # here only a few SymPy-backed quantifier expression can be constructed from a general interface.
         # Operation is limited to sum and product, and constrain can only be a range.
-        raise NotImplementedError()
+        raise FutureWarning("Not Implemented")
```

Comment:
Otherwise, PyCharm gives "new_quantifier_expression not implemented" warnings to all subclasses.